nurpv.sh

```diff
@@ -182,25 +182,31 @@ nurpvlademodus(){
 						openwbDebugLog "MAIN" 1 "pv ladung beendet"
 						rm ramdisk/nurpvoff
 					else # Keine aktive Ladung erkannt, Mindestüberschuss unterschritten
-						if [[ $cpunterbrechungmindestlaufzeitaktiv == 1 ]]; then # Mindestwartezeit für Ladestopp nach CP Unterbrechung aktiviert	
+						if [[ $cpunterbrechungmindestlaufzeitaktiv == "1" ]]; then # Mindestwartezeit für Ladestopp nach CP Unterbrechung aktiviert	
```

Comment:
Doppelte "[" interpretieren reguläre Ausdrücke, hier wird jedoch einfach ein Vergleich von Strings benötigt. Die Variable sollte dabei auch immer in Anführungszeichen gekapselt werden.
`if [ "$cpunterbrechungmindestlaufzeitaktiv" == "1" ]; then`

Reply:
done.